### PR TITLE
Add test-things repo

### PR DESCRIPTION
**Pull request by netomi**
This PR add the test-things repo as requested in https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3426.

Please comment on this ticket if you want the description / topics set to specific values.

### CODE REVIEW

**Comment by egekorkan** [comment]
Could you add the following:
- Description: Collection of Things that can be used for testing different IoT protocols, security mechanisms and interaction styles
- Topics: Internet of Things, Web of Things, Protocols, Testing
- Website: thingweb.io

CC @danielpeintner

**Comment by netomi** [comment]
updated PR with the requested metadata.

**Review by danielpeintner — APPROVED** [review]
LGTM

**Comment by netomi** [comment]
change applied, repo has been created.

File: otterdog/eclipse-thingweb.jsonnet [hunk 1]
```diff
@@ -68,6 +68,19 @@ orgs.newOrg('eclipse-thingweb') {
         },
       ],
     },
+    orgs.newRepo('test-things') {
+      allow_update_branch: false,
+      description: "Collection of Things that can be used for testing different IoT protocols, security mechanisms and interaction styles",
+      homepage: "http://thingweb.io",
```
Comment (heurtematte) [inline otterdog/eclipse-thingweb.jsonnet:74]: Avoid http protocol

Comment (netomi) [reply otterdog/eclipse-thingweb.jsonnet:74]: ty, I was just following the convention from another repo, but I will use https in the future for all homepages and also adapt existing ones.

**Pull request merged**